Orange/classification/svm.py

```diff
@@ -24,6 +24,14 @@ def __init__(self, C=1.0, kernel='rbf', degree=3, gamma="auto",
         super().__init__(preprocessors=preprocessors)
         self.params = vars()
 
+    def __call__(self, data, progress_callback=None):
+        if len(np.unique(data.Y)) > 1:
+            return super().__call__(data, progress_callback)
+        else:
+            warnings.warn("Single class in data, returning Constant Model.")
+            maj = MajorityLearner()
+            return maj(data)
```

Comment:
We have to be careful if we move this check to the `Learner` base class, because we probably expect a specific model as a result in some cases. E.g. logistic regression models can be connected to nomograms and might crash if all of a sudden they happen to be constant models (without coefficients etc).
But I still think this is not an svm-specific issue.